lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -55,14 +55,15 @@ export const migrateGranuleRecord = async (
     { name, version }
   );
 
-  // Schema validation on Dynamo record will fail if `record.execution`
-  // does not exist
-  const executionCumulusId = await executionPgModel.getRecordCumulusId(
-    knex,
-    {
-      url: record.execution,
-    }
-  );
+  // It's possible that very old records could have this field be undefined
+  const executionCumulusId = record.execution
```

Comment:
I'm not sure that I follow. 

The reason for this change is that previously, we had been doing schema validation on the Dynamo granule record, and since `execution` is required as of the current Dynamo schema, that means the record would either fail the validation or would have an execution.

But since we removed that Dynamo schema validation across the board, I realized we needed to clean up the code here to not assume that an `execution` exists for a granule. Since they could have old granule records from before `execution` was required in the Dynamo schema.

The confusing thing is that `execution` from the Dynamo granule records is/should be a link to a Step Function execution URL, so that's how we have to look for the execution record in PG

Reply:
Sorry, I was trying to recall/presuming that 

`   // It's possible that very old records could have this field be undefined ` 

was the reason the value also wasn't required in the postgres schema.   Thanks for the clarification on the other points.